src/did/src/sign_strategy.rs

```diff
@@ -95,6 +95,13 @@ impl TransactionSigner for TxSigner {
             Self::ManagementCanister(signer) => signer.sign_transaction(transaction).await,
         }
     }
+
+    async fn sign_digest(&self, digest: [u8; 32]) -> Result<Signature> {
```

Comment:
would it be possible to add a unit test for a Local signer to show the entire loop DIGEST->SIGN->VERIFY

Reply:
Done